src/interactivity/base/InteractivityFactory.cpp

```diff
@@ -330,19 +335,15 @@ using namespace Microsoft::Console::Interactivity;
                 // will return the console handle again, not the owning
                 // terminal's handle. It's not entirely clear why, but WS_POPUP
                 // is absolutely vital for this to work correctly.
-                const auto windowStyle = WS_OVERLAPPEDWINDOW | WS_POPUP;
-                const auto exStyles = WS_EX_TOOLWINDOW | WS_EX_TRANSPARENT | WS_EX_LAYERED | WS_EX_NOACTIVATE;
-
-                // Attempt to create window.
-                hwnd = CreateWindowExW(exStyles,
+                hwnd = CreateWindowExW(WS_EX_TOOLWINDOW | WS_EX_NOACTIVATE,
                                        reinterpret_cast<LPCWSTR>(windowClassAtom),
                                        nullptr,
-                                       windowStyle,
+                                       WS_POPUP,

```

Comment:
No `WS_OVERLAPPEDWINDOW` = no DWM, no border, no size. Makes it cheaper and easier. Seems to still work too.

Reply:
Wonder if this fixes the "openconsole has a 0x0 window at 0x0" problem